apps/service-portal/src/components/Sidebar/NavItem/NavItem.tsx

```diff
@@ -49,20 +53,32 @@ const NavItemContent: FC<Props> = ({
 
   const badgeActive: keyof typeof styles.badge = badge ? 'active' : 'inactive'
 
+  const animatedIcon = icon
+    ? `./assets/icons/sidebar/${icon.icon}.svg`
+    : undefined
+
   return (
     <Box
       className={[
         styles.navItem,
         styles.navItemActive[navItemActive],
         collapsed && 'collapsed',
         'navitem',
+        isDashboard && styles.dashboard,
       ]}
       display="flex"
       alignItems="center"
       justifyContent={collapsed ? 'center' : 'spaceBetween'}
       cursor={showLock ? undefined : 'pointer'}
       position="relative"
-      onClick={showLock ? (hasArray ? undefined : onClick) : onClick}
+      onClick={() => {
+        const id = icon && iconIdMapper(icon.icon)
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        const a: any = id && document.getElementById(id)
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Could this `: any` be a `HTMLElement` type and we could get rid of the eslint disabling?